src/corelibs/U2View/src/ov_assembly/AssemblyReadsAreaHint.cpp

```diff
@@ -66,20 +66,31 @@ AssemblyReadsAreaHint::AssemblyReadsAreaHint(QWidget* parent)
     setObjectName("AssemblyReadsAreaHint");
 }
 
-static QString getCigarString(const QString& ci) {
+static QString getCigarString(const QString& ci, bool trimmed = false) {
```

Comment:
The method is over-designed. 
It has a single use in the code, so you do not need different states (trimmed/not-trimmed) for it: it is dead code.

You can just rename it to getCigarStringForTooltip() and always do the trimming.

Reply:
Yep, really, didn't notice